README.md

```diff
@@ -30,3 +30,23 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+The if statement is just constant time so 1. The else statment does n/3 recursively 3 times so 3T(n/3). Count is constant so add 1. The outter most for loop runs $n^2$ times, the first nested loop runs n times and the inner most for loop runs $n^2$ times. So you get $n^2 * n * n^2 = n^5$ after you disregrard the constant times from the if statement and the var count.
+
+So, to start we have:<br> $T(n) = 1$, <br> $3T(n/3) + n^5$ , <br> $3(3T((n/3)/3) + ( (n/3)^5) + n^5$ = $9T(n/9) + 3(n/3)^5 +n^5$. <br> Then we get $log_3n$ from the three levels of recursion $\frac{n}{3^i} = 1$ <br> 
+1 because the recursion happens until the size equals 1. <br>
+$n=3^i$<br>
+$i=log_3n$
+So, sub in $i$ for the unfolding<br>
+$3^iT(n/3^{i}) + \sum_{k=0}^{i-1} 3^{i}\* (n/3^i)^5$ and use summation for the count increasing for the nested loop that happens $n^5$ times.
```

Comment:
Can you show some more details here please? How did you get to this sum?

Reply:
Please show more detail here. Also, your sum variable is $k$ but you're not using it...